java-checks/src/main/java/org/sonar/java/checks/MembersDifferOnlyByCapitalizationCheck.java

```diff
@@ -108,8 +108,10 @@ private static boolean isValidIssueLocation(Symbol currentMember, Symbol knownMe
     return !sameOwner(currentMember, knownMember) || isOverriding(knownMember) || getDeclarationLine(currentMember) > getDeclarationLine(knownMember);
   }
 
-  private static boolean bothPublic(Symbol s1, Symbol s2) {
-    return s1.isPublic() && s2.isPublic();
+  private static boolean sameVisibilityNotPrivate(Symbol s1, Symbol s2) {
+    return (s1.isPublic() && s2.isPublic())
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1067)